runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -127,5 +169,135 @@ private void setDodgyCharges(int dodgyCharges)
 	{
 		this.dodgyCharges = dodgyCharges;
 		config.dodgyNecklace(dodgyCharges);
+		if (currentInfoBoxIDs.get(EquipmentInventorySlot.AMULET) == ItemID.DODGY_NECKLACE)
+		{
+			if (dodgyCharges > 0)
+			{
+				createJewelleryInfobox(ItemID.DODGY_NECKLACE, dodgyCharges, EquipmentInventorySlot.AMULET);
+			}
+			else
+			{
+				removeJewelleryInfobox(EquipmentInventorySlot.AMULET);
+			}
+		}
+	}
+
+	@Subscribe
+	public void onItemContainerChanged(ItemContainerChanged event)
+	{
+		if (event.getItemContainer().getItems().length > 14)
+		{
+			return;
+		}
+
+		Item[] items = event.getItemContainer().getItems();
+
+		int amulet = -2;
+		int bracelet = -2;
+		int ring = -2;
+		if (items.length >= 3)
```

Comment:
You should early return on < 3

Reply:
It's in the else case

Reply:
Early returning reduces block indent overall